Add meta description for tracing and eval pages (#13134)
mlflow/mlflow: docs/source/llms/tracing/index.rst

.. meta::

>   :description: MLflow Tracing is a feature that enables LLM observability in your apps. You can automatically log traces for LangChain, LlamaIndex, and more.

Author: ```suggestion
  :description: MLflow Tracing is a feature that enables LLM observability in your apps. MLflow automatically logs traces for LangChain, LlamaIndex, and more.
```
